src/NHSD.BuyingCatalogue.Solutions.API/Controllers/Suppliers/SuppliersController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Solutions.API.ViewModels.Suppliers;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.Controllers.Suppliers
+{
+    [Route("api/v1/suppliers")]
```

Comment:
This can now use the [controller] key word. Please review to a similar route that uses the name of the controller to define the route.

Reply:
👍